src/logcollector/read_mysql_log.c

```diff
@@ -22,11 +22,13 @@ void *read_mysql_log(logreader *lf, int *rc, int drop_it) {
     size_t str_len = 0;
     int need_clear = 0;
     char *p;
-    char str[OS_MAXSTR + 1];
-    char buffer[OS_MAXSTR + 1];
+    char str[OS_MAXSTR - OS_LOG_HEADER];
```

Comment:
Why are we substracting `OS_LOG_HEADER` here?

Reply:
Because the `w_msg_hash_queues_push` function has a buffer inside it in which the buffer sent with a header will be concatenated. The maximum size of the header is defined by the tag.

Reply:
Got it. Same as with the other PRs, consider defining a new macro for this value :)